samples/fabric/go-cli/Makefile

```diff
@@ -0,0 +1,9 @@
+all: clean build
+
+.PHONY: clean
+clean:
+	rm -rf ./bin
+
+.PHONY: build
+build:
+	go build -o ./bin/fabric-cli fabric-cli.go
```

Comment:
Considering you have `make build` here to build the binary, should the binary file `./bin/fabric-cli` be deleted from the repo?

Reply:
We will keep the binary on the repo, so that it's not necessary for the users who ever is only testing (without any modifications) can continue to use the binary. And the `Makefile` is only to help update the binary for development purpose.